README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
```

Comment:
Note: Tetris is a tile-matching video game that was created by Russian software engineer Alexey Pajitnov in 1984 for the Electronika 60.